features/steps/command/shell.feature

```diff
@@ -72,7 +72,7 @@ Feature: Running shell commands
       Scenario: Running zsh commands
         When I run the following commands with `zsh`:
         \"\"\"bash
-        echo "Hello \c"
+        echo -e "Hello \c"
```

Comment:
I think it's more a problem of test isolation. We/I wrongly assumed some shell - bash, zsh - configuration. But this may interfere with ones shell's setup - zshrc, bashrc.

I think it makes more sense, to explilcitly use a configuration for each shell to make sure, they're configured correctly.

Options I see:
1. Add a wrapper for `sh`, `bash`, `zsh` which uses it's only configuration file instead using the one of the user
2. Set the path to the configuration file via environment variable

What do you think?